### DISCUSSION TITLE
Urodzinowy meetup - zaproszenie

### DISCUSSION THREAD

**Discussion by mgrabarz** [AzureCommunityPL/group#169] [Wydarzenia]
Cześć,
W tym roku 9.12 mija dokładnie 5 lat od pierwszego meetupu naszej grupy, planujemy więc urodziny!
Oto link do nagrania z 2015 roku: https://www.youtube.com/watch?v=XoVPKFopTJQ&feature=youtu.be

Tym razem spotykamy się na Teamsach, by każdy mógł włączyć kamerkę, porozmawiać i czasem coś pokazać na swoim ekranie :) 

**Nie planujemy żadnej technicznej sesji, ale za to pobawimy się w Q&A. Oto zasady:**
- Zadajemy pytania związane z Azure.
- Możecie wskazać dowolną osobę (spośród całego community!) jako odpowiadającego.
- Jeżeli tej osoby nie będzie na spotkaniu, organizatorzy postarają się na nie sami odpowiedzieć.
- **Można zgłosić dowolną ilość pytań tutaj:  https://azurecommunitypl.typeform.com/to/BZ7cuI9S**

Na GitHub dalej zbieramy Wasze pomysły, co ma się pojawić na urodzinach - link tutaj: https://github.com/AzureCommunityPL/group/discussions/156 Może uważasz, że powinniśmy zorganizować panel dyskusyjny, wirtualny szampan, wspomnienia? Czekamy na Twoje pomysły!

![5_urodziny_maugp_720](https://user-images.githubusercontent.com/20926456/100888839-d2c8cf80-34b6-11eb-9425-9f6655b7aace.png)

Zapisy: https://www.meetup.com/Microsoft-Azure-Users-Group-Poland/events/274942770/
Przypominajka do kalendarza: https://add.eventable.com/events/5eda719d28e23c04a87af3ba/5fc6a97da76fce0414008d83

Do zobaczenia!

**Comment by eklime** [comment]
Chciałbym przypomnieć o ankiecie. Obecnie wygląda trochę "smutnie". Nik nie ma żadnych pytań?
![image](https://user-images.githubusercontent.com/12485979/101149613-792fe480-361f-11eb-8479-15fa8592d335.png) 

Dla przypomnienia link: https://azurecommunitypl.typeform.com/to/BZ7cuI9S

**Reply by aryll** [reply]
@eklime i jak z tymi pytaniami wyszło - będą dzisiaj jakieś? 😄

**Reply by eklime** [reply]
Istny szał - 5 pytań :D
